pages/views/_layouts/base.njk

```diff
@@ -14,7 +14,9 @@
   {# Additional site meta information #}
   {% block meta %}{% endblock meta %}
   {# Site icon #}
+  <link rel="icon" type="image/x-icon" href="{{ '/assets/logo-light-blue.svg' | url }}" />
   <link rel="icon" type="image/png" href="{{ '/assets/logo-64x64.png' | url }}" />
+  <link rel="icon" type="image/png" href="{{ '/assets/logo-120x120.png' | url }}" />
```

Comment:
Not sure about placing icons without any definitions regarding their params...
Is it according to the favicons guide?